### PR TITLE
fix strategy test for api and linting

### PR DESCRIPTION
**Pull request by Coniferish**
### Summary 
Closes unstructured-api issue [188](https://github.com/Unstructured-IO/unstructured-api/issues/188)
The test and gist were using different versions of the same file (jpg/pdf), creating what looked like a bug when there wasn't one. The api is correctly using the `strategy` kwarg.

#### Checkout to `main`
- Comment out the `@pytest.mark.skip` decorators for the `test_partition_via_api_with_no_strategy` test
- Add an API key to your env:
  - Add `from dotenv import load_dotenv; load_dotenv()` to the top of the file and have `UNS_API_KEY` defined in `.env`

- Run `pytest test_unstructured/partition/test_api.py -k "test_partition_via_api_with_no_strategy"`
^the test will fail

#### Checkout to this branch 
- (make the same changes as above)
- Run `pytest test_unstructured/partition/test_api.py -k "test_partition_via_api_with_no_strategy"`

### Other
`make tidy` and `make check` made linting changes to additional files

### CODE REVIEW

File: test_unstructured/partition/test_api.py [hunk 1]
```diff
@@ -140,13 +136,14 @@ def test_partition_via_api_with_no_strategy():
...
+    # elements_hi_res[3].text =
+    #     'LayoutParser: A Uniﬁed Toolkit for Deep Learning Based Document Image Analysis'
+    # while elements_no_strategy[3].text = ']' (as of this writing)
     assert elements_no_strategy[4].text != elements_hi_res[4].text
 
 
-@pytest.mark.skipif(skip_outside_ci, reason="Skipping test run outside of CI")
-@pytest.mark.skipif(skip_not_on_main, reason="Skipping test run outside of main branch")
+# @pytest.mark.skipif(skip_outside_ci, reason="Skipping test run outside of CI")
+# @pytest.mark.skipif(skip_not_on_main, reason="Skipping test run outside of main branch")
```
Comment (cragwolfe) [inline test_unstructured/partition/test_api.py:146]: i think we still want these skips, i.e. don't hit the live API frequently in tests

Comment (Coniferish) [reply test_unstructured/partition/test_api.py:146]: Whoops. Thanks

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
 
 ### Fixes
 
+* **Fix api test** A test seemed to show that the api was not correctly applying the `strategy` kwarg when called, but it was a mistake with the file type and this verifies that the api is working as expected.
```
Comment (cragwolfe) [inline CHANGELOG.md:11]: please remove the CHANGELOG item. the policy is to only include changes that impact end users of the unstructured library in this file.

Comment (Coniferish) [reply CHANGELOG.md:11]: Thanks. Wasn't positive on that.

**Comment by Coniferish** [comment]
@cragwolfe would it be worth investigating why the same file in different formats is partitioned into different elements? `.jpeg` file partitioned via `auto` returned 17 elements while `.pdf` returned 25

File: test_unstructured/partition/test_api.py [hunk 3]
```diff
@@ -140,8 +136,9 @@ def test_partition_via_api_with_no_strategy():
     elements_hi_res = partition_via_api(filename=filename, strategy="hi_res", api_key=get_api_key())
 
     # confirm that hi_res strategy was not passed as default to partition by comparing outputs
-    # FIXME(crag): elements_hi_res[4].text is 'sacon oot barvard o', the fast output.
-    # should be 'Harvard University {melissadell,jacob carlson}@fas.harvard.edu' (as of writing)
+    # elements_hi_res[3].text =
+    #     'LayoutParser: A Uniﬁed Toolkit for Deep Learning Based Document Image Analysis'
+    # while elements_no_strategy[3].text = ']' (as of this writing)
```
Comment (cragwolfe) [inline test_unstructured/partition/test_api.py:141]: nit: just noting the index in the comment doesn't match the index in the comparison below, which is slightly awkward.

**Review by cragwolfe — APPROVED** [review]
LGTM!

File: test_unstructured/partition/test_api.py (hunk 3 above)
Comment (Coniferish) [reply test_unstructured/partition/test_api.py:141]: Yup, just changed. Ty

**Pull request merged**